src/main/java/models/GameEngine.java

```diff
@@ -195,6 +195,9 @@ private void playerMenu() {
             case 3:
                 System.out.println("what card you like to attack with");
                 int attackCard=sc.nextInt();
+                if(checkIfTapped(currentPlayer.getTableCards().get(attackCard - 1))){
```

Comment:
Kan vi få in en sout typ "card is tapped" eller något så man vet varför man åker tillbaks till menyn? ;P